docs/Inputs.md

```diff
@@ -125,6 +125,16 @@ However, in some cases (e.g. inside a `<ReferenceInput>`), you may not want the
 <AutocompleteInput source="gender" choices={choices} translateChoice={false}/>
 ```
 
+
+By default the component matches choices with the current input searchText, if it finds a match this choice will be selected. For example, using the `choices`: `[{id:'M',name:'Male',id:'F',name:'Female'}]` and the user enters the text `male` then the component will set the input value to `M`. Using the `inputValueMatcher` prop the component allows you to change how choices are matched. For example, given the choices: `[{id:1,iso2:'NL',name:'Dutch'},{id:2,iso2:'EN',name:'English'},{id:3,iso2:'FR',name:'French'}]` you can create the following `inputValueMatcher` to match choices on the iso2 code: 
+```javascript
```

Comment:
Can you add an empty line before this one ? Otherwise, the markup for code won't be displayed